### DISCUSSION TITLE
「如果今天面試官是機器人」

### DISCUSSION THREAD

**Discussion by wu51000** [1101-GS3073-C/Python-Community-GS3073-C#732] [General]
「如果今天面試官是機器人」
我們從申請大學入學開始了我們面試的第一扇門
而當面試官由真人轉變成ai後，透過觀察面試者臉部的微表情與應答進行評分。
然而這對於平時溝通較無情緒起伏或臉部表情癱瘓的人來說是非常不利的
https://www.cheers.com.tw/article/article.action?id=5096862&page=3

**Comment by HsuChen88** [comment]
有好處也有壞處，好處就是不會再因為長相而被扣分，壞處就是不能再因為長相而加分。

**Comment by yjpxi** [comment]
如果面試官是ai的話，我覺得可以縮短面試時間還可以加速招聘的過程，但是有些面試問題是需要情感的理解，ai對於情感理解這方面一定有困難。

**Comment by unnamed0604** [comment]
我認為面試的重點應該在於面試者和面試官的互動狀況，而非單單在檢測面試者的答覆及表情，我不認為AI能真的像面試官那樣針對個別面試者進行互動並判斷面試者的能力與心性，但若面試只是設計成單純的回答問題，那我想AI是能勝任的。
